repl/src/test/scala/ammonite/repl/ScriptTests.scala

```diff
@@ -271,7 +271,7 @@ object ScriptTests extends TestSuite{
       }
       'caching{
 
-        def createTestInterp(storage: Storage) = new Interpreter(
+        def createTestInterp(storage: Storage, predefS: String = "") = new Interpreter(
```

Comment:
Get rid of this `predefS` argument and just use `ammonite.repl.Main.defaultPredefString` directly